docs/_reporting/hubs/configure-scopes.md

```diff
@@ -32,30 +32,57 @@ FinOps hubs uses Cost Management exports to import cost data for the billing acc
   _Microsoft Cost Management does not support managed exports for Microsoft Customer Agreement billing accounts. Please [configure Cost Management exports manually](#️-configure-exports-manually)._
 </blockquote>
 
-For the most seamless experience, we recommend allowing FinOps hubs to manage exports for you when possible. This is the simplest to setup and requires the least effort to maintain over time. We only recommend configuring Cost Management exports manually if you cannot grant FinOps hubs access to manage exports for you.
+For the most seamless experience, we recommend allowing FinOps hubs to manage exports for you when possible. This option requires the least effort to maintain over time.
+
+<!-- TODO: The  sentence above can be changed when ME can be configured through UI and not command line -->
```

Comment:
Is this about "simplest to setup"? If so, I think that's a fair point. But I don't think we need the comment unless you're going to comment out the sentence you want to revert back to later. Either way, I think the simpler sentence is totally fine without the "simplest to setup" part. The ultimate point is conveyed and we probably don't need this comment.
```suggestion
```